backend/tests/api/api_test.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_health_check_response_healthy(mocker):
-    mock_test_connection = mocker.patch("src.api.test_connection", return_value=True)
+    mock_test_connection = mocker.patch("src.api.api.test_connection", return_value=True)

```

Comment:
Should we rename the directory or the file so that they are not the same?

Reply:
yeah that is a good point, maybe it should be called `app.py` as app is what comes from it. I'll change to that, open to other ideas though